backend/backend-standard-and-best-practices.md

```diff
@@ -213,7 +214,19 @@ Most commonly used are:
 
 You can read more about this and other status codes in [this link](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status).
 
-### 6- Routes
+## 6- API Documentation
+
+There are lots of different tools in the market to document APIs. Here in Wolox we decided to use Openpi 3 (A.K.A Swagger) as a standard documentation tool in all of our techs. The reasons behind this decision were some of the advantages OpenApi prodvides:
+
+* **Quick to use**: The properties that you need to complete in every endpoint are always the same and there are only a few. You can also frequently reuse components and the generic information is simple to complete.
+* **Wide range of available features**: From describing parameters, making authentication and authorization schemes, to even using the generated documentation of an endpoint to mock test the endpoint in question.
+* **Intuitive and easy to use interactive UI**: Swagger up a view where show how to do a request and how to response the API. 
```

Comment:
I think you missed a few words here

![](http://cdn-webimages.wimages.net/04f1b5454338612504654a464c6f92ad122856-wm.jpg?v=3)